src/components/DiscountItem/DiscountItem.css

```diff
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+*{
+ box-sizing: border-box;
+ padding: 0;
+
+}
```

Comment:
these 7 lines must be inside of index.scss
because they are global styles, and in this file you keep styles for your component only